plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -514,6 +514,29 @@ public void testDateYearOfEraPredicate()
                 "Conversion failed when converting date and/or time from character string\\.");
     }
 
+    @Override
+    public void testNativeQuerySimple()
+    {
+        // override because SqlServer provides an empty String as the name for unnamed column
```

Comment:
String -> string
SqlServer -> SQL Server

is empty string a bad thing? why?

Reply:
I think we don't want to have a column with empty name. It wouldn't be accessible in the surrounding SQL. I think that instead we could handle such column as "anonymous". However I'd have to check the spec. From what I recall, it might be required that a table function returns only named columns.
```
PTFs use PTF descriptor areas for the following purposes:
(...) 3) To describe the row type of the result.
...
The describe component procedure populates the initial result row type descriptor to describe the proper result columns of the PTF. 
...
The column name is in the NAME component. This is a variable length character string (...)
```
so per spec, it should not be possible that a table function produces an anonymous column. I think though that this is worth consideration as an extension.

Reply:
> I think we don't want to have a column with empty name. It wouldn't be accessible in the surrounding SQL

Indeed, empty column name is not allowed in Trino.

> so per spec, it should not be possible that a table function produces an anonymous column. I think though that this is worth consideration as an extension.

makes sense. let's not spend time on this within this PR

thanks for explaining.